service/docs/source/install.rst

```diff
@@ -0,0 +1,92 @@
+
+Install
+=======
+
+Continuous integration creates packages for a variety of Linux
+distributions each time certain branches hosted on Github are updated.
+This is supported by Github Actions, in conjunction with the OpenSUSE
+Build Service.  The ``dev`` branch, which is the default branch
+containing the most up to date stable development, is published here:
+
+    https://download.opensuse.org/repositories/home:/geopm/
+
+Until the version 2.0.0 version of GEOPM is tagged, packages
+reflecting the latest ``release-v2.0-candidate`` branch will be
+published here:
+
+    https://download.opensuse.org/repositories/home:/geopm:/release-v2.0-candidate/
+
+The packages built from this service are named:
+
+- ``geopm-service``
+- ``geopm-service-devel``
+- ``libgeopmd0``
+- ``python3-geopmdpy``
+
```

Comment:
What about dasbus?  Aren't we building some of our dependencies in OBS?
They should be listed here.